pkg/sql/types/types.go

```diff
@@ -1066,7 +1131,7 @@ func (t *T) Name() string {
 //   int4[]       _int4
 //
 func (t *T) PGName() string {
-	name, ok := oid.TypeName[t.Oid()]
+	name, ok := oidext.TypeName(t.Oid())
```

Comment:
I don't have a better suggestion right now, but I'm nervous that people will forget to call the oidext version when they're supposed to... anything we can do about this? Maybe we just want to wrap the oid package with our own oid package and ban imports of the other oid package?